### DISCUSSION TITLE
请问 PAGSolidLayer 如何使用

### DISCUSSION THREAD

**Discussion by KingOfSu** [Tencent/libpag#2615] [Q&A] [answered]
查看 https://pag.io/apis/android/reference/org/libpag/PAGSolidLayer.html
这里面 PAGSolidLayer 的公共构造方法是：

public PAGSolidLayer (long nativeContext)

在issues上也没有看到相关问题。所以想请问下这个生成构造有啥办法吗？

应用场景：切换深浅色的时候图标颜色也需要更改，现在是深色和浅色用了两套问题，如果我们可以使用这个 PAGSolidLayer 的话那么我们直接填充颜色就可以了

**Comment by kevingpqi123** [comment, marked as answer]
PAGSolidLayer 不支持自己创建，需要设计师在 AE 中添加 SolidLayer，导出成 pag 文件终端才能使用，让你们的设计师基于 AE 的 SolidLayer 依据产品需求设计相关素材
